src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -91,42 +104,43 @@ extends AbstractMap[Key, Value]
     table = new Array[Entry](newSize)
     mask = newSize - 1
     oldTable.foreach( entry =>
```

Comment:
Actually, 2.12 won't be able to fully optimize `Array.foreach`: even after inlining the implementation and eliminating the closure, every value is boxed and unboxed before it's passed to the consumer. Logged here: https://github.com/scala/scala-dev/issues/27#issuecomment-216119276